From 06a149fea50be0d9afc442946b48c564c4d79f38 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 19 Mar 2024 17:30:18 +0000
Subject: [PATCH 0979/1085] ARM: dts: bcm2712-cm5: i2c_csi_dsi is i2c_csi_dsi1

Make i2c_csi_dsi an alias for i2c_csi_dsi1, and remove the spurious
redefinition in the cm5 .dts files.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm4io.dts | 2 --
 arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm5io.dts | 2 --
 arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi      | 2 +-
 3 files changed, 1 insertion(+), 5 deletions(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm4io.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm4io.dts
@@ -3,8 +3,6 @@
 
 #include "bcm2712-rpi-cm5.dtsi"
 
-i2c_csi_dsi: &i2c_csi_dsi0 { }; // An alias for compatibility
-
 // The RP1 USB3 interfaces are not usable on CM4IO
 
 &rp1_usb0 {
--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm5io.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5-cm5io.dts
@@ -3,8 +3,6 @@
 
 #include "bcm2712-rpi-cm5.dtsi"
 
-i2c_csi_dsi: &i2c_csi_dsi0 { }; // An alias for compatibility
-
 / {
 	__overrides__ {
 		i2c_csi_dsi = <&i2c_csi_dsi>, "status";
--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
@@ -249,7 +249,7 @@ i2c_csi_dsi0: &i2c6 { // Note: This is f
 i2c_csi_dsi1: &i2c0 { // Note: This is for MIPI1 connector
 };
 
-i2c_csi_dsi: &i2c_csi_dsi0 { }; // An alias for compatibility
+i2c_csi_dsi: &i2c_csi_dsi1 { }; // An alias for compatibility
 
 cam1_reg: &cam0_reg { // Shares CAM_GPIO with cam0_reg
 };
